test/test.js

```diff
@@ -210,6 +210,24 @@ const createTestSuite = (useElectronNet) => {
       })
     })
 
+    if (useElectronNet) {
+      // for some reason, Node.js parses the header value differently
+      // so this test doesn't work in node, only in electron
+      it('should reject with error when headers contain invalid symbols', function () {
+      // This test somehow fails 80% of the time in CI...
+      // probably because the test matrix overloads the remote server or something?
+      if (process.env.CI) return this.skip()
```

Comment:
```suggestion
        // This test somehow fails 80% of the time in CI...
        // probably because the test matrix overloads the remote server or something?
        if (process.env.CI) return this.skip()
```